psl-cli/src/main/java/org/linqs/psl/cli/Launcher.java

```diff
@@ -198,7 +199,9 @@ private Database runInference(Model model, DataStore dataStore, Set<StandardPred
             outputGroundRules(inferenceApplication.getGroundRuleStore(), path, false);
         }
 
-        inferenceApplication.inference();
+        boolean commitAtoms = parsedOptions.hasOption(CommandLineLoader.OPTION_SKIP_ATOM_COMMIT_LONG) ? false : true;
```

Comment:
In general ternary operators are usually the wrong choice.
(I love them, but there is usually a better way.)

Just negate `parsedOptions.hasOption()`.